libtransact/src/workload/batch_gen.rs

```diff
@@ -26,15 +26,15 @@ use protobuf::Message;
 
 use crate::error::{InternalError, InvalidStateError};
 use crate::protocol::batch::{Batch as ProtocolBatch, BatchBuilder, BatchPair};
-use crate::protocol::transaction::Transaction as ProtocolTransaction;
+use crate::protocol::transaction::Transaction;
+use crate::protos::batch::Batch;
```

Comment:
We should maybe call this ProtobufBatch here too since we have the protocol Transaction and usually Batch/Transaction work together.

Reply:
Fixed